ramls/dataExport.raml

```diff
@@ -44,3 +44,79 @@ traits:
           body:
             text/plain:
               example: "Internal server error"
+  /fileDefinitions:
+    displayName: File upload service
+    description: Method to create file definition
+    post:
+      description: File definition
+      is: [validate]
+      body:
+        application/json:
+          type: fileDefinition
+      responses:
+        201:
+          body:
+            application/json:
+              type: fileDefinition
+        400:
+          description: "Bad request"
+          body:
+            text/plain:
+              example: "Bad request"
+        422:
+          description: "Unprocessable Entity"
+          body:
+            application/json:
+              type: errors
+        500:
+          description: "Internal server error"
+          body:
+            text/plain:
+              example: "Internal server error"
+    /{fileDefinitionId}:
+      get:
+        description: Method to get File definition by id
+        responses:
+          200:
+            body:
+              application/json:
+                type: fileDefinition
+          400:
+            description: "Bad request"
+            body:
+              text/plain:
+                example: "Bad request"
+          404:
+            description: "Not found"
+            body:
+              text/plain:
+                example: "Not found"
+          500:
+            description: "Internal server error"
+            body:
+              text/plain:
+                example: "Internal server error"
+      post:
```

Comment:
Isn't it better to use a separate endpoint for upload? It's a little deceiving that GET  /filedefinitions/{fileID} will get file definition and POST will do an upload of a file
we can do POST /filedefinitions/{fileID}/upload for file uploads